data_conditions.md

```diff
@@ -13,16 +13,7 @@ The data made available through INTERMAGNET are provided for your use and are no
 
 ## Acknowledgement of data from observatories participating in INTERMAGNET
 
-INTERMAGNET publishes its data in the form of Digital Object Identifiers (DOIs). Definitive INTERMAGNET data publications from 2013 onwards are available as DOIs:
-
-- [https://doi.org/10.5880/INTERMAGNET.2013](https://doi.org/10.5880/INTERMAGNET.2013)
-- [https://doi.org/10.5880/INTERMAGNET.2014](https://doi.org/10.5880/INTERMAGNET.2014)
-- [https://doi.org/10.5880/INTERMAGNET.1991.2015](https://doi.org/10.5880/INTERMAGNET.1991.2015)
-- [https://doi.org/10.5880/INTERMAGNET.1991.2016](https://doi.org/10.5880/INTERMAGNET.1991.2016)
-- [https://doi.org/10.5880/INTERMAGNET.1991.2017](https://doi.org/10.5880/INTERMAGNET.1991.2017)
-- [https://doi.org/10.5880/INTERMAGNET.1991.2018](https://doi.org/10.5880/INTERMAGNET.1991.2018)
-- [https://doi.org/10.5880/INTERMAGNET.1991.2019](https://doi.org/10.5880/INTERMAGNET.1991.2019)
-- [https://doi.org/10.5880/INTERMAGNET.1991.2020](https://doi.org/10.5880/INTERMAGNET.1991.2020)
+INTERMAGNET publishes its data in the form of Digital Object Identifiers (DOIs). Definitive INTERMAGNET data publications from 2013 onwards are available as [DOIs](/new_data_download.html#downloading_data_using_dois).
```

Comment:
I think this link needs changing (to /data_download.html) doesn't it?

Reply:
Yes indeed